lib/main.js

```diff
@@ -100,6 +109,10 @@ function tokenExchangeHandler({
   };
 }
 
+async function _cacheIssue({cache, key, fn}) {
```

Comment:
It doesn't seem like this helper is really saving us anything, it's only called once and it's a one-liner. Can we just call `cache.memoize({key, fn})` in its one place instead?

Reply:
Updated.